### PR TITLE
Vc9

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @djhoese and @dopplershift this should help us to keep `vc9` alive for a while longer. If you identify another package that should be listed here please let me know.

@isuruf can you review this one and merge if it is OK?

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -316,9 +316,11 @@ gst_plugins_base:
   - 1.12  # [linux]
   - 1.14.0  # [osx]
 gdal:
-  - 2.3
+  - 2.3  # [not (win and vc9)]
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]: I think you need `vc==9`

Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: Thanks!

**Comment by ocefpaf** [comment]
Let's give this a go. Thanks @isuruf for the review!

**Comment by dopplershift** [comment]
Can you think of a reason why `conda smithy rerender` wouldn't change anything after this PR?
```
Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
Adding in variants from internal_defaults
Adding in variants from /Users/rmay/miniconda3/conda_build_config.yaml
Re-rendered with conda-smithy 3.1.12 and pinning 2018.12.04
No changes made. This feedstock is up-to-date.
```
Looking at our 2.7 windows build I still see the config for geos 3.7 and proj4 5.2:
```
> cat .ci_support/win_c_compilervs2008cxx_compilervs2008python2.7vc9.yaml                    [rerender]
c_compiler:
- vs2008
channel_sources:
- conda-forge,defaults
channel_targets:
- conda-forge main
cxx_compiler:
- vs2008
geos:
- 3.7.0
numpy:
- '1.11'
pin_run_as_build:
  geos:
    max_pin: x.x.x
  proj4:
    max_pin: x.x.x
  python:
    min_pin: x.x
    max_pin: x.x
  vc:
    max_pin: x
proj4:
- 5.2.0
python:
- '2.7'
vc:
- '9'
zip_keys:
- - python
  - vc
  - c_compiler
  - cxx_compiler
```

**Pull request merged**